docs/src/content/docs/ru/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,290 @@
+    				'./src/tailwind.css',
+    			],
+    		}),
+    		tailwind({
+    			// Отключите базовые стили:
+    			applyBaseStyles: false,
+    		}),
+    	],
+    });
+    ```
+
+5.  Добавьте плагин Starlight Tailwind в `tailwind.config.cjs`:
+
+    ```js ins={2,7}
+    // tailwind.config.cjs
+    const starlightPlugin = require('@astrojs/starlight-tailwind');
+
+    /** @type {import('tailwindcss').Config} */
+    module.exports = {
+    	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    	plugins: [starlightPlugin()],
+    };
+    ```
+
+### Стилизация Starlight с использованием Tailwind
+
+Starlight будет использовать значения из вашей [конфигурации темы Tailwind](https://tailwindcss.com/docs/theme) в его UI.
+
+Если установлены, следующие параметры переопределят стили Starlight по умолчанию:
+
+- `colors.accent` — используется для ссылок и выделения текущего элемента;
+- `colors.gray` — используется для цветов фона и границ;
+- `fontFamily.sans` — используется для текста UI и контента;
+- `fontFamily.mono` — используется для примеров кода.
+
+```js {12,14,18,20}
+// tailwind.config.cjs
+const starlightPlugin = require('@astrojs/starlight-tailwind');
+const colors = require('tailwindcss/colors');
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// Ваш предпочтительный акцентный цвет. Индиго наиболее близок к стандартным настройкам Starlight.
+				accent: colors.indigo,
+				// Ваш предпочтительный оттенок серого. Цинк наиболее близок к стандартным настройкам Starlight.
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				// Ваш предпочтительный шрифт для текста. По умолчанию Starlight использует системные шрифты.
+				sans: ['"Atkinson Hyperlegible"'],
+				// Ваш предпочтительный шрифт для кода. По умолчанию Starlight использует системные моноширинные шрифты.
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Темизация
+
+Цветовая тема Starlight может быть настроена путем переопределения её стандартных кастомных свойств.
+Эти переменные используются по всему UI, причём различные оттенки серого используются для текста и цветов фона,
+а акцентный цвет используется для ссылок и выделения текущих элементов в навигации.
+
+### Редактор цветовой темы
+
+Используйте ползунки ниже, чтобы изменить палитры акцентного и серого цветов Starlight.
+Темные и светлые области предпросмотра будут показывать результирующие цвета, и вся страница также обновится, чтобы показать ваши изменения.
+
+Когда вы довольны внесенными изменениями, скопируйте CSS или код Tailwind ниже и используйте его в вашем проекте.
+
+import ThemeDesigner from '../../../../components/theme-designer.astro';
+
+<ThemeDesigner
+	labels={{
+		presets: {
+			label: 'Заготовки',
+			ocean: 'Ocean',
+			forest: 'Forest',
+			oxide: 'Oxide',
+			nebula: 'Nebula',
```

Comment:
I guess it was intentional to not translate these four names? Happy not to if you think it’s best, but in other languages we did translate them.

Reply:
Yes, it was intentional. Okay, I translated it